### ISSUE TITLE
***.Designer.cs for .resx file doesn't update after .resx file modification

### ISSUE THREAD

**Issue by ELU-DEV-LEAD** [dotnet/ResXResourceManager#537]
When I add a key to a resx file using the Visual Studio extension, the linked *.Designer.cs file isn't updated to reflect the change.

Visual Studio 2022 Version 17.5.1
ResXManager VSIX version 1.74.5652.0 

To Reproduce:
1. In solution browser, right click `foo.resx`, Open With ResXManager
2. Click plus to add new key and enter key name e.g. `bar`
3. Close ResXManager
4. Check `foo.Designer.cs`, no `bar` entry added

Troubleshooting steps attempted without resolution:
- Run Visual Studio with elevated administrative privileges
- Reinstall VSIX extension

Behavior confirmed with colleagues.

Basically the same behavior as #525 but for VSIX version.

**Comment by tom-englert** [comment]
I can't reproduce:
![devenv_wdMF7FuwZg](https://user-images.githubusercontent.com/10229954/222156852-4c8a733b-b1c5-487e-8606-a237e25d064e.gif)

Are you sure the custom tool is set correctly?
Does it work when you use the built-in resource editor?

**Comment by ELU-DEV-LEAD** [comment]
Oh shoot, I thought I had checked every detail!
Apologies for the wasted effort @tom-englert 

Opening the .resx in the Managed Resource Editor revealed that it had switched from Access Modifier: public to No code generation for some reason. Changing it to public once more fixed the behavior.

Maybe this will at least help other people. When migrating your project to the newest .NET version, double-check that this is not set to No code generation:
![NoCodeGen](https://user-images.githubusercontent.com/118987923/222195716-3ad90127-7822-4db8-8789-826c972c5db5.png)
